Access API/AccessAPI/SignalR/SignalRHandler.cs

```diff
@@ -46,9 +44,9 @@ public async Task SendGroupMessage(string groupName, string messageTag, string m
                                 JsonConvert.DeserializeObject<TestRequest>(message))));
```

Comment:
Check warning siger "Possible null reference argument for parameter 'request' in 'TestResponse SuggestorSimulator.GenerateTestResponse(TestRequest request)'."